docs/book/getting-started/deploying-zenml/deploy-with-docker.md

```diff
@@ -44,7 +44,8 @@ The following environment variables can be passed to the container:
 * **ZENML\_SERVER\_RATE\_LIMIT\_ENABLED**: This variable controls the rate limiting for ZenML API (currently only for the `LOGIN` endpoint). It is disabled by default, so set it to `1` only if you need to enable rate limiting. To determine unique users a `X_FORWARDED_FOR` header or `request.client.host` is used, so before enabling this make sure that your network configuration is associating proper information with your clients in order to avoid disruptions for legitimate requests.
 * **ZENML\_SERVER\_LOGIN\_RATE\_LIMIT\_MINUTE**: If rate limiting is enabled, this variable controls how many requests will be allowed to query the login endpoint in a one minute interval. Set it to a desired integer value; defaults to `5`.
 * **ZENML\_SERVER\_LOGIN\_RATE\_LIMIT\_DAY**: If rate limiting is enabled, this variable controls how many requests will be allowed to query the login endpoint in an interval of day interval. Set it to a desired integer value; defaults to `1000`.
-
+* **ZENML\_SERVER\_JWT\_SECRET\_KEY**: If you want to make sure your session tokens remain valid when spinning servers down and up again, a constant JWT secret key should be set. Feel free to choose a random string. Change this value to invalidate all existing issued sessions.
```

Comment:
This is already documented a bit further down in the `Advanced server configuration options` section